src/components/MediaBrowser/MediaBrowserData.js

```diff
@@ -26,6 +26,7 @@ function MediaBrowserData({ docType }) {
   const _searchParams = new URLSearchParams({
     q: searchTerm,
     [TYPES]: searchType,
+    sort: searchTerm ? null : 'created_desc',
```

Comment:
If you could use the constants for these searchParams it will help with maintainability and keeping track of where these are used. 
`sort:` -> `[SORT]:`
`'created_desc'` -> `SORT_CREATED_DESC`
There is a full list constants for searchParam and their possible values in src/common/constants/searchParams.js